oolab/src/test/java/agh/ics/oop/RectangularMapTest.java

```diff
@@ -0,0 +1,27 @@
+package agh.ics.oop;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class RectangularMapTest {
```

Comment:
to jest ok jak najbardziej, ale przydałby się jeszcze test całego silnika, tak jak w poleceniu